EIPS/eip-5792.md

```diff
@@ -122,26 +136,41 @@ Returns the status of a call batch that was sent via `wallet_sendCalls`. The ide
 type GetCallsParams = string;
 
 type GetCallsResult = {
-  status: 'PENDING' | 'CONFIRMED';
+  batchId: `0x${string}`;
+  chainId?: `0x${string}`;
+  batchStatus: string; // See "Status Codes"
   receipts?: {
     logs: {
       address: `0x${string}`;
       data: `0x${string}`;
       topics: `0x${string}`[];
     }[];
     status: `0x${string}`; // Hex 1 or 0 for success or failure, respectively
-    chainId: `0x${string}`;
     blockHash: `0x${string}`;
     blockNumber: `0x${string}`;
     gasUsed: `0x${string}`;
     transactionHash: `0x${string}`;
   }[];
+  capabilitiesData?: Record<string, any> | undefined;
```

Comment:
I find it kinda confusing that this is the same phrase as on the request, but would likely be different data?

Reply:
I fully support renaming this parameter. `capabilitiesResultsData` is a bit long, how about just `capabilitiesResults` or `capabilitiesStatus`?

Reply:
I prefer clarity even if verbose so I think I like `capabilitiesResultsData`

Reply:
> I prefer clarity even if verbose so I think I like `capabilitiesResultsData`

Done.